kinto/plugins/history/__init__.py

```diff
@@ -0,0 +1,17 @@
+from kinto.core.events import ResourceChanged
+
+from .listener import on_resource_changed
+
+
+def includeme(config):
+    config.add_api_capability('history',
```

Comment:
The history is activated for every buckets. Spawn an issue to add a setting in order to limit the buckets where it's enabled ?